ungleich/dynamicweb
# Task/4799/show selected vms only in calculator

To test,

1. In the admin dashboard (Home › Datacenterlight › Vm pricings), open the VMPricing instance associated with your calculator instance

2. Choose the VM templates in the "Vm templates to show" option

3. Save and verify in the front end that the templates chosen in 2 only appear in the calculator
